h/static/styles/variables.scss

```diff
@@ -151,3 +151,15 @@ $score-width: 40px;
 $score-height: $score-width;
 $top-bar-height: 40px;
 $input-border-radius: 2px;
+
+// Variables for the new colour palette
```

Comment:
(Changing the grays to use the new palette everywhere would _definitely_ be part of a separate card I should add. Needs discussion with @conordelahunty as well about how we split that up).